python_scripts/xx_basic_parameters_tuning.py

```diff
@@ -1,20 +1,22 @@
 # %% [markdown]
 # # Introduction to scikit-learn: basic model hyper-parameters tuning
 #
-# In this lecture note, we aim at:
-# * illustrate the influence of changing model parameters;
-# * illustrate how to tune these hyper-parameters;
-# * evaluate the model performance together with hyper-parameters tuning.
+# The process to learn a predictive model is driven by a set of internal
+# parameters and a set of training data. These internal parameters are called
+# hyper-parameters and are specific for each family of models. In addition,
+# a set of parameters are optimal for a specific dataset and thus they need
+# to be optimized.
 #
-# ## Recall of basic preprocessing and model fitting
+# This notebook shows:
+# * the influence of changing model parameters;
+# * how to tune these hyper-parameters;
+# * how to evaluate the model performance together with hyper-parameters
+#   tuning.
 #
-# In the previous notebook, we show how to preprocessed different type of data
-# and integrate this preprocessing in a machine learning pipeline containing a
-# predictor.
+# ## Recall of a basic preprocessing and predictive model fitting
 #
-# We will recall this example. First, we will load the data and organize it
-# into a `data` and a `target` variable. The ultimate goal is to train a
-# predictor able to estimate the wages from different censing data.
+# In the previous notebook, we show how to build a predictive model containing
+# a preprocessing step to deal with heterogeneous data.
```

Comment:
Is this really necessary? I would just remove this section to keep the notebook concise and focused.